src/Http/Response.php

```diff
@@ -1137,6 +1137,22 @@ public function type($contentType = null)
         return $contentType;
     }
 
+    /**
+     * Sets a content type definition into the map.
+     *
+     * E.g.: setType('xhtml' => ['application/xhtml+xml', 'application/xhtml'])
+     *
+     * This is needed for RequestHandlerComponent and recognition of types.
+     *
+     * @param string $type Content type.
+     * @param string|array $mimeType Definition of the mime type.
+     * @return void
+     */
+    public function setTypeMap($type, $mimeType)
```

Comment:
Is this method setting a map or just a single type? The name and implementation don't appear to match.

Reply:
A single one. Is setTypeToMap() better?
I didnt want to addType() as it would also overwrite existing one.

Reply:
Hmm. On second thought I don't have a better name that isn't long.

Reply:
`setTypeMap()` suggests that it sets whole map of types, not single type.

Main keyword here is _map_. After look at [dictionary](https://en.oxforddictionaries.com/definition/map), I think it could be `mapType()`, `associateType()`, `linkType()`.

Reply:
I also don't think ``setTypeMap()`` for setting one tpye within the map is a good name.
The best would really be ``mapType()``, unfortunately.

How about ``setTypeMapped()`` or ``setMappedType()``?
Or ``updateType()``?

``associateType()``, isn't bad either.